packages/core/core/src/core/app-configuration/config-loader.ts

```diff
@@ -59,18 +60,22 @@ export default (dir: string) => {
       return acc;
     }
 
-    // Note: we don't have access to strapi logger at this point so we can't check if
     if (!VALID_EXTENSIONS.includes(extensionLower)) {
-      warnSkippedConfig(
+      logWarning(
         `Config file not loaded, extension must be one of ${VALID_EXTENSIONS.join(',')}): ${
           file.name
         }`
       );
       return acc;
     }
 
+    // restricted names are also restricted from being prefixes
+    const res = RESTRICTED_FILENAMES.find((restrictedName) =>
```

Comment:
That might not be the clearest variable name I can think of :sweat_smile: 

Could we rename that to something clearer? Also, is there a reason why this is defined this high compared to the actual check?

Reply:
Oops, fixed!